rpm/containers-common.spec

```diff
@@ -27,6 +27,10 @@
 %define netavark_epoch 2
 %endif
 
+%if %{defined fedora} || (%{defined rhel} && 0%{?rhel} == 9)
```

Comment:
Also should this say rhel < 10? I guess we no longer build on 8 and older but logically this makes more sense then special casing a single version IMO.